### ISSUE TITLE
[Bug] PhantomJS Hangs

### ISSUE THREAD

**Issue by MichielDeMey** [ncopa/docker-phantomjs-alpine#2]
Using PhantomJS on Alpine will currently hang.
See https://github.com/ariya/phantomjs/issues/14186 for more information.

**Comment by sylus** [comment]
Yeah ran across the same issue with this. It looks to me like an issue with pselect6. This is reaching but I wonder if we are running across this same issue.

Strace (linked from other issue): https://github.com/ariya/phantomjs/files/211444/phantomjs.strace.txt

https://patchwork.ozlabs.org/patch/559138/ (Use linux pselect syscall when available)

**Comment by LaurentGoderre** [comment]
@ncopa Do you have any idea for how to fix this. I am struggling with this too.
